en-US/Characters_Codex/03_Creating_a_Character/Creating_a_Character.md

```diff
@@ -1,6 +1,6 @@
 # Creating a Character
 
-As a player, begin by creating a character on the character sheet in the back of this book or a color-coded one at <mark>[LINK]</mark>. <!-- Link to be determined -->
+As a player, begin by creating a character on the character sheet in the back of this book or a color-coded one at <mark>Free5e.com</mark>. <!-- Link to be determined -->
```

Comment:
The `<mark>...</mark>` will cause this to be highlighted (normally with a yellow background). Did you intentionally leave that in?
Also, this should probably be a link. If we want to keep the highlighting:

```suggestion
As a player, begin by creating a character on the character sheet in the back of this book or a color-coded one at [<mark>Free5e.com</mark>](https://free5e.com).
```
If we don't want the highlighting:

```suggestion
As a player, begin by creating a character on the character sheet in the back of this book or a color-coded one at [Free5e.com](https://free5e.com).
```